lib/events/Header/homeHeaderPostEvents.js

```diff
@@ -1,5 +1,15 @@
 if (Meteor.isClient) {
 
+	
+function btnSearch () {
```

Comment:
Function declarations should not be placed in blocks. Use a function expression or move the statement to the top of the outer function.